llvm/lib/CodeGen/SelectionDAG/DAGCombiner.cpp

```diff
@@ -12644,6 +12644,8 @@ SDValue DAGCombiner::foldPartialReduceMLAMulOp(SDNode *N) {
           TLI.getTypeToTransformTo(*Context, LHSExtOpVT)))
     return SDValue();
 
+  EVT ResultVT = N->getValueType(0);
```

Comment:
The extraction of ResultVT appears to be a separable NFC.  Please separate and land, and then rebase.